spec/model/user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'test name', photo: 'test photo', bio: 'test bio', post_counter: 0) }
+
+  before { subject.save }
+
+  it 'name should be present' do
```

Comment:
- Add validation tests to the models. This PR is about validations and testing validations makes sense.
- For example,
`describe 'validations' do
  it { should validate_presence_of(:name) }
end
`

Reply:
Hello, @ afanimartin, thanks for the review, But I do have a concern, my code contains model validations which are being tested in the spec files and these tests are working perfectly because of the presence of the validations in the respective models. So I think I do not need to use a different method